spikeinterface/toolkit/qualitymetrics/pca_metrics.py

```diff
@@ -11,7 +11,7 @@
 _possible_pc_metric_names = ['isolation_distance', 'l_ratio', 'd_prime', 'nearest_neighbor']
```

Comment:
You need to add 'nn_isolation' here.
So it is tested and integrated magically in the table.